README.md

```diff
@@ -1,27 +1,32 @@
-# Project Name
+# Fcc Cat Photo App
 
-> One paragraph statement about the project.
+> A project using Basic HTML in styling of Cat Photos
 
-![screenshot](./app_screenshot.png)
+![cat photo](image.png)
```

Comment:
I would use a more descriptive alt text, such as `Cat photo app screenshot` since it's a screenshot or a similar descriptive text. Also, why don't we rename the file as `app_screenshot.png`?